assets/stylesheets/style.scss

```diff
@@ -224,6 +224,7 @@
     // WCC Components
     @import '../../client/components/shipping/packages/style';
     @import '../../client/components/indicators/style';
+		@import '../../client/components/text/style';
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/398/commits/1e76910535dc441ee89832ea26ffd36057bd6fff